Sources/Testing/SourceAttribution/Backtrace.swift

```diff
@@ -76,7 +76,16 @@ public struct Backtrace: Sendable {
         initializedCount = .init(clamping: backtrace(addresses.baseAddress!, .init(clamping: addresses.count)))
       }
 #elseif os(Android)
-#if !SWT_NO_DYNAMIC_LINKING
+#if compiler(>=6.3)
+      if #available(Android 33, *) {
+        initializedCount = addresses.withMemoryRebound(to: UnsafeMutableRawPointer.self) { addresses in
+          .init(clamping: backtrace(addresses.baseAddress!, .init(clamping: addresses.count)))
+        }
+      }
+      else {
+        print("Backtraces are unavailable on Android API 32 and older.")
```

Comment:
@grynspan, what do you think of this warning?

Reply:
It's not necessary (and we wouldn't use `print()` for this anyway.) You can leave it out.